job_definitions/notify_buyers_to_award_closed_briefs_4_weeks.yml

```diff
@@ -0,0 +1,73 @@
+{% set environments = ['production'] %}
+---
+{% for environment in environments %}
+- job:
+    name: "notify-buyers-to-award-closed-briefs-4-weeks-{{ environment }}"
+    display-name: "Notify buyers to award closed briefs (4 weeks) - {{ environment }}"
+    project-type: freestyle
+    description: "Send first email notification to buyer users to award closed briefs, after 4 weeks."
+    parameters:
+      - string:
```

Comment:
If we change param orders in our script, don't forget to also change them in here

Reply:
Done